images/users/hasura.svg

```diff
@@ -0,0 +1 @@
7IVdWqt9cCyZceX4qeHOx Na09vlx8XK6inW/26Xyqc4Jra0OqpW+aIapL2yNvE9NcOm8osE9yYhqQFYSoQAOopW89IaoC1I1/ 8B7z3N5e84Pnjpgf7Xrb9PbHX9paKVwtVRpuRfn+riit3rxo9G2crW0VauqmtrryZn+/eauvzwsp e0+e9EJW0dtKAase3xlTpoz9EX/q8dnvG+djLWVDWbX9tUGwQlW9TZo9NvwC1jwcf0uXLjXbt2+P 1GItjZbgAADkkUtVE3kb8POuWS+6yJltrKrjVatWJfZ7qlrQ1da5a9asMQ899JDzx4v2kQstKcu5 sLaeBtBdrNBVsKvA0qVqZw3uu1qFSuiQH5o84NqkD50/161bl9r5m+daslzqSKBqVM5nKDpCVKAB 5reNVZI9eiRaAKQK1B/sOGJ+45LZZnFXp7m1f755etdRL0x9eldj/3h3rUovKBua2tuZpwLKhvzs 0Z+lm35uVpQGwQp6VVGrClUFqklVqebt+LvwwgvNxz72MXPttdeao0ePmg984AORvg9VqAAA1M+1 Ab88Vie5NMCWdPs5BW0uBoCi8E1tfZMKkeOgAPpzn/uck4/Nheemji9Xqyv1uFT95sLzXcfRJz7x CWePcxcCeeR3X6bZ0telSR55fZ5psp0rIaquoV3sQgA0EiEq0CAr6ghR5bEdh70QVTqntHjv66aK VBuoPvdqbyIVqlnkhYDTp6cSmuaNKl9t219RZa1tV5x2qOqaJYtmmus+8knz/muvMwsWLBj/+Be+ 8IVI308BqiZhAACA+ri0plNWWq2G5VJ1rQYedUtqWytoczVEFVVMaWDZ1coRVcu62K5WXNhmrk+y uOmmm8wjjzyS+rnM5eMoC/sR4c4LLq5VnFZLX5da+eb1mkrHnCvXGayLChjTzCYAGuPSTmMu64xe UaaAVO18y9lA9a7fWWge+i8Xem/nzWov1La1Id/KhQvNXYsXmwdG/1j581/8RW+dz/80dy4Basy0 PbVdtX2/dPnl5ku/8ivmD97zHq9qtZ71WbNM4amef3/+mQ+aD3/0UxMCVM0Q3bZtW6Tv+9G5VKEC ABAHlypR89jKV1wL7JIc9EtzHbqgFHS5toafKPhyOYB2parJ5eoqWwGaZktr148j1/ch8nHeT+s5 4FIlal7bzLp0/tC1hKtLGACNQogKNNDKOfV9/TeePFDzPgpz/uIj53rhah6pLa8NTFdfeKEX4Ck0 1fvXj37swpjXN0WAfXIqVNVaqgqvv/7+93thtv6vYDVLbYyjWLyw05u8oOfcGb/8kUmf37RpU6QL Tk26oAoVAIB4uDSLPs/rSrk06JdkcK7KF9f3o64/XQpS9XgUfLm6fqV9bmq9WxcsW7bM6eNLx9WK FSsafnxl4Thy7VhCPFxsZ2pb+jaSqnFdCtSYmNa4cz5QZCQNQANdPUOhyIjZNxjt61WNqjVQFZRW M3dWm/mL3z3X3L7ppZp//Ovix8UZRQpC7bqcCk7t+2kGpG3Nzaazrc20t7SY3sFBc+LkSTM8MuL0 MTdl9LHOaG/3Hmf/0FDDHrNtozx+rP3n/2xm/Pqvm507d5qf79plukcvvJ/71381//vHP3Zyu+kP Xq1p2t3dXbVlz3lzO7wA1Ts+Zsw10866ZNIfNQpRo/jomVShAgAQB9va1RV5rk5yqa1e0gN+2o+u DyrqWlRB19q1a1NdI1XPP1Uuur69XKo002NRW2bXz60K6tetW9eQbZeV40hYPzB/bDDu2vhZo1v6 ulSFKnltm61jzaUW0trvtChHkRGiAg2m9pxf7I4eYj3w+L6aIaoo3Ln1188avf8bFe+jkOiv/uqv vPe3b9/uXQwq5LJvxf6/Ef7TvHleRaNLlYtTW1vHb81N74RaCicVRiqUPDowYAaHh515zHqcCnun j95am99pOKCPiR5r3+jjVhDcqMfdfsYZ3kXgFVdc4d3sMfe/b7656tf9wsUXm58lUDliQ1L7PLAX qGrDa/+v4/6aa66p+n0++8Gzx6u+5105eWCqnirUS4vZGRkAgNi5tpZTnitRXfrdkh7oVSi5cePG TOwXhXEaaFeY2sh9pOtgbSPdstAK0KUQ1VY7ZyGov/XWW71tpzA1qepLHUPr16/PTEvJLLT8RrT9 6loVtB6Pzu2Nqnx26ZyU5+spUZWtKyGqS8tiAGkgRAUafdE1u8l87UD0atT9PQPmBzuOeOug1vLb l88xT7/0tlfB6kfrNKrSTsGRDbaWLl1a9Q8kG65WfHyHDnk3a87cueaMM88c//99oxd31UKxtANU hY4KSG1wWo0NK3U7OTzsBaqNDCajPm5V1La1t49XqOpxJ12lOuWMM3yPp1ru8Jl9/cKp46dz6lRz fsnao0EGIhaEuP9jjz1W9TH+3tVzvckKMu1dl5jp51wx6fejChUAgPS5NPCj65E8t3h0ra2e9n1S lRNZCbkshcqqSlWFnG5JVpToOlgD+wq+XKoCD7I/XaL95Ho1qqWWojrGNLlAt7jOczqOFJ5m5TgS HUcuVeUjPmqz7WIraT3/GlX97FIlal5b+ZaeS1xZ+9m1CYlAoxGiAimotxpVa6MuWTQj0LqnCno+ /+ruip9/8MEHzV133RXo59pKwmp+vm+fd6tkamfl8ro3+/pS2R8KFae1tXnhY1tztKWiFWLOKAkm FUrqporPJEJV/by2U8FpRx2PuzQIltJA9WSMj7tz4cJJH6sVyP9ChQvi9576+Ozp0837LrggseNC z42Kv8/oc0+TFKy410KlChUAgPi4FHLlfcDPtVZvGvRL8jEpLNL6jFmiAEA3hTyqqtItjm2kkEuD 66p4dWXQNwwXKwezFKKK/vZR4KmbHrsCpyjbVedsHaM6jrIUnpaeF5BPOp5dbenbqBCVStTG0bIB Op+6cn7Xvs/7NgcqIUQF0rjwiqEa9dvPHvQC0loWd3WaxQs7K1aj
```

Comment:
Thanks for your PR.

Could you please run your SVG through svgo in order to optimize it.

Reply:
@xtuc - I have actually run through svgo already. But the optimization seems to be minimal.

Reply:
That looks strange to me, the SVG is just a wrapper around a Base64 encoded image. Could you please re-export it as a real svg? Otherwise it won't be responsive.